lib/main.dart

```diff
@@ -1,7 +1,7 @@
 import 'package:lavenir/models/user.dart';
-import 'package:lavenir/screens/shifts/add-availability.dart';
-import 'package:lavenir/screens/shifts/request-shift.dart';
-import 'package:lavenir/screens/shifts/view-shifts.dart';
+import 'package:lavenir/screens/shifts/availability.dart';
+import 'package:lavenir/screens/shifts/announcement.dart';
```

Comment:
Could we rename this to "announcements"? It seems like a dumb detail but when we actually code out the announcement list we'll probably want to name the individual announcement widget the singular